src/components/become-tutor-steps/firstStep/FirstStep.js

```diff
@@ -0,0 +1,75 @@
+import { Box, Checkbox, FormControlLabel, Typography } from '@mui/material'
+import { useTranslation } from 'react-i18next'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import img from '~/assets/img/mentor-home-page/become-tutor/first-step.png'
+import { styles } from './first-step.styles'
+
+const FirstStep = ({ data, handleChange, handleBlur, errors }) => {
```

Comment:
FirstStep is not an appropriate name for the component.

Reply:
You should sync with Rostyslav PR with children's buttons.

Reply:
sure